pkg/systemd/init.js

```diff
@@ -510,9 +510,11 @@ $(function() {
     var template_template = $("#service-template-tmpl").html();
     mustache.parse(template_template);
 
-    var unit_actions = [                          // <method>:<mode>
+    var unit_primary_actions = [                          // <method>:<mode>
         { title: _("Start"),                 action: 'StartUnit' },
-        { title: _("Stop"),                  action: 'StopUnit' },
+        { title: _("Stop"),                  action: 'StopUnit' }
```

Comment:
Just a hint: Personally I'm a big fan of trailing commas in such lists, as this avoids this kind of diff noise. We use these in plenty of places, so feel free to do it here, too.